app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -14,15 +14,16 @@ def github
 
   def failure
     error = request.env["omniauth.error"]
+    class_name = error.present? ? error.class.name : ""
 
     DatadogStatsClient.increment(
       "omniauth.failure",
       tags: [
-        "class:#{error}",
+        "class:#{class_name}",
         "message:#{error&.message}",
```

Comment:
Do we need a `try` here as well?

Reply:
No, because all exceptions in Ruby have `.message`